### PR TITLE
Recipe

### PR DESCRIPTION
**Pull request by shaaibu7**
- [ ] Should display a list of recipes created by the logged-in user as in the wireframe.
- [ ] Should lead to recipe details.
- [ ] If the user is the owner of the recipe, should allow the user to delete it.

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+  def index
+    @recipes = current_user.recipes
+  rescue NoMethodError
+    redirect_to new_user_session_path
+  end
 
-  def show; end
+  def show
+    @recipe = current_user.recipes.find(params[:id])
+  end
```
Comment (rica213) [inline app/controllers/recipes_controller.rb:10]: - [ ] How about we handle errors here as well ❓ ❔  For example If a user tries to access a recipe that doesn't belong to them by manipulating the URL parameters, we redirect them to the recipe index instead. 
Though I am the one who is working on the Recipe details, so maybe I should be the one who needs to implement it? 
What do you think? 🧠

**Review by rica213 — CHANGES_REQUESTED** [review]
# 🟡 STATUS: REQUIRED CHANGES ♻️ 
Hi Shaaibu

You have been doing a great job so far. I am impressed 💯 🥇 
Kindly give me your opinion on what I asked in the comment section 👇🏽 👇🏽 



Happy coding!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (shaaibu7) [reply app/controllers/recipes_controller.rb:10]: Hi Larissa, 

Since you are working on the show page you can handle the exception on your end.

**Review by rica213 — APPROVED** [review]
# APPROVED 🟢 

Hi Shaaibu,

You have done a great job implementing the Recipe List 💯 👍🏽 🥇 
Thank you for that. Now we can move to another task 🗒️ 
Let's merge it 🛸 🛸 



😁 😁

**Pull request merged**